algorithms/ar-expsq/csharp/ExponentiationBySquaring.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+public class ExponentiationBySquaring
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6352723)